libbpf-cargo/src/gen.rs

```diff
@@ -248,6 +248,76 @@ fn gen_skel_map_defs(
         )
     };
 
+    write!(
+        skel,
+        r#"
+        pub struct {struct_name}<'a> {{
+            inner: &'a  {inner_ty},
+        }}
+
+        impl<'a> {struct_name}<'a> {{
+        "#,
+        inner_ty = inner_ty,
+        struct_name = struct_name
+    )?;
+
+    for map in MapIter::new(object) {
+        if map_is_mmapable(map) {
+            continue;
+        }
```

Comment:
hm... why this check? the fact that map is mmapable has nothing to do with whether we should have accessor for it or not. @danobi, I see similar check above, why are we skipping mmapable maps. They might be .data/.rodata/.bss, in which case it's still generally useful to have map accessor for them. But BPF_F_MMAPABLE can be specified by users explicitly, in which case omitting such maps would be bad as well.

Reply:
This is how it was done in the original skeleton generation logic. What do you think about replacing this with a check against map name (i.e. does it end in a `.bss`, `.data`, `.rodata`)?

Reply:
This is original skeleton generation logic: https://github.com/torvalds/linux/blob/master/tools/bpf/bpftool/gen.c#L380-L404 :) let's keep them in sync

Reply:
> This is original skeleton generation logic: https://github.com/torvalds/linux/blob/master/tools/bpf/bpftool/gen.c#L380-L404 :) let's keep them in sync

Haha, that's a fair point! Just meant this wasn't anything new I added :)

Reply:
@anakryiko So you think we should just remove the check entirely then? So the .bss map would be included in both skel.maps().bss() and skel.bss() for example.

Reply:
I vote yes, I don't like when tool make arbitrary decisions on what should or shouldn't be available to users. Those internal maps are still maps, so should be represented in maps().

Reply:
This makes sense to me. @danobi any thoughts?

Reply:
Sure, yeah. Add them. Don't remember why I originally did this but it was probably b/c I thought it was being mmap'd so no need to expose the actual maps.